### PR TITLE
[DOC] replace gitter mentions by appropriate links, references

### PR DESCRIPTION
**Pull request by TonyBagnall**
as per the agreement at CC meeting on 8/3/22, I have purged all references I could find to Gitter.

### CODE REVIEW

File: docs/source/developer_guide/git_workflow.rst [hunk 1]
```diff
@@ -96,6 +96,6 @@ GitHub, clone, and develop on a new branch.
 .. note::
 
    If any of the above seems like magic to you, look up the `Git documentation <https://git scm.com/documentation>`_.
-   If you get stuck, chat with us on `Gitter`_ or `Discord`_.
+   If you get stuck, chat with us on `Discord`_.
```
Comment (fkiraly) [inline docs/source/developer_guide/git_workflow.rst:99]: I would say, primarily slack, on the `#contributors` channel. We are much more active on slack imo.

File: docs/source/get_involved/contributing.rst [hunk 2]
```diff
@@ -32,11 +32,10 @@ and people who are looking to learn and develop their skills.
    Research <https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html>`_ to get started and learn more about open-source collaboration.
 
 .. _discord: https://discord.com/invite/gqSab2K
-.. _gitter: https://gitter.im/sktime/community
 
 .. note::
 
-   If you get stuck, chat with us on `Gitter`_ or `Discord`_.
+   If you get stuck, chat with us on `Discord`_.
```
Comment (fkiraly) [inline docs/source/get_involved/contributing.rst:38]: slack as well.

**Review by fkiraly — CHANGES_REQUESTED** [review]
I would actually point to slack for anything developer related.

I would also add references to the community collaboration sessions on Fridays, where appropriate.

**Comment by TonyBagnall** [comment]
> I would actually point to slack for anything developer related.
> 
> I would also add references to the community collaboration sessions on Fridays, where appropriate.

I'm happy for you to add anything you want, I was just trying to action the removal of gitter, that is all this PR is meant to do

**Comment by fkiraly** [comment]
> I'm happy for you to add anything you want, I was just trying to action the removal of gitter, that is all this PR is meant to do

Well, if we're already doing it, imo, let's do it "right" and funnel new contributors towards slack primarily.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 contact_links:
   - name: "\U0001F4AC All other questions and general chat"
-    url: https://gitter.im/sktime/community
-    about: Chat with the sktime community on Gitter
+    url: https://discord.com/invite/gqSab2K
```
Comment (fkiraly) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
    url: https://join.slack.com/t/sktime-group/shared_invite/zt-62i7aejn-vXc3nOWF26S_P3VXFPWisQ
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+    about: Chat with the sktime community on Discord
```
Comment (fkiraly) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    about: Chat with the sktime community on Slack
```

File: docs/source/get_involved/contributing.rst [hunk 5]
```diff
@@ -32,11 +32,10 @@ and people who are looking to learn and develop their skills.
    Research <https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html>`_ to get started and learn more about open-source collaboration.
 
 .. _discord: https://discord.com/invite/gqSab2K
```
Comment (fkiraly) [inline docs/source/get_involved/contributing.rst:34]:
```suggestion
.. _discord: https://discord.com/invite/gqSab2K
.. _slack: https://join.slack.com/t/sktime-group/shared_invite/zt-62i7aejn-vXc3nOWF26S_P3VXFPWisQ
```

File: docs/source/get_involved/contributing.rst (hunk 2 above)
Comment (fkiraly) [inline docs/source/get_involved/contributing.rst:38]:
```suggestion
   If you get stuck, chat with us on `Slack`_, or join one of the community sessions on `Discord`_.
```

File: docs/source/developer_guide/git_workflow.rst (hunk 1 above)
Comment (fkiraly) [inline docs/source/developer_guide/git_workflow.rst:99]:
```suggestion
   If you get stuck, chat with us on `Slack`_, or join one of the community sessions on `Discord`_.
```

File: docs/source/get_involved/contributing.rst [hunk 6]
```diff
@@ -127,4 +126,4 @@ All contributions are recorded in
 
 .. note::
 
-   If we have missed anything, please `raise an issue <https://github.com/alan-turing-institute/sktime/issues/new/choose>`_ or chat with us on `Gitter`_ or `Discord`_.
+   If we have missed anything, please `raise an issue <https://github.com/alan-turing-institute/sktime/issues/new/choose>`_ or chat with us on `Discord`_.
```
Comment (fkiraly) [inline docs/source/get_involved/contributing.rst:129]:
```suggestion
   If we have missed anything, please `raise an issue <https://github.com/alan-turing-institute/sktime/issues/new/choose>`_ or chat with us on Slack`_.
```

File: docs/source/get_involved/mentoring.rst [hunk 7]
```diff
@@ -43,4 +43,4 @@ How to apply
 
 We accept applications on a rolling basis and we will be in touch with you shortly!
 
-If you have any questions or get stuck, please don't hesitate to `chat with us <https://gitter.im/sktime/community>`_ or email us at info@sktime.org.
+If you have any questions or get stuck, please don't hesitate to `chat with us <https://discord.com/invite/gqSab2K>`_ or email us at info@sktime.org.
```
Comment (fkiraly) [inline docs/source/get_involved/mentoring.rst:46]:
```suggestion
If you have any questions or get stuck, please don't hesitate to `chat with us on slack <https://join.slack.com/t/sktime-group/shared_invite/zt-62i7aejn-vXc3nOWF26S_P3VXFPWisQ>`_, join on of our collaborative community sessions on `discord <https://discord.com/invite/gqSab2K>`_, or email us at info@sktime.org.
```

**Review by fkiraly — CHANGES_REQUESTED** [review]
I've made explicit suggestions how to change this to "slack primary" pointers.

You just need to accept the suggestions, and all shall be fine (I hope), so no additional work on you.

**Comment by fkiraly** [comment]
> I'm happy for you to add anything you want,

Do you mind accepting the suggestions? They replace discord by slack whenever developer content is the context.

**Comment by fkiraly** [comment]
oops, something is broken now, let me check

**Comment by fkiraly** [comment]
?? seems like there is no error message, readthedocs build just didn´t start. Just restart it?

**Review by fkiraly — APPROVED** [review]
Restarted readthedocs, looks good.

**Review by fkiraly — CHANGES_REQUESTED** [review]
Actually, no:

the links in `contributing.rst` don´t point anywhere: discord is missing, and slack has the wrong capitalization. Let me fix that.

**Review by fkiraly — APPROVED** [review]
I think, all fixed now.

**Pull request merged**